00-HTML-CSS-basics/exerc1-v.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form</title>
+  </head>
+  <body>
+    <section>
+      <h2>Sign up Form</h2>
+      <form method="post">
+        <p><label>First Name: <input name="firstname" required></label></p>
```

Comment:
`<p>` common use is to wraph text, so in this case is better to use just `<div>` to stack the inputs